src/geo/transform.js

```diff
@@ -23,8 +23,8 @@ class Transform {
         this.tileSize = 512; // constant
 
         this._renderWorldCopies = renderWorldCopies === undefined ? true : renderWorldCopies;
-        this._minZoom = minZoom || 0;
-        this._maxZoom = maxZoom || 22;
+        this.minZoom = minZoom || 0;
+        this.maxZoom = maxZoom || 22;
```

Comment:
Does get minZoom() still return the correct value, since that function returns this._minZoom not this.minZoom? I may be missing something, but why is this change necessary?